packages/core/src/Stores/client-store.js

```diff
@@ -1253,6 +1253,29 @@ export default class ClientStore extends BaseStore {
         }
     }
 
+    @action.bound
+    fetchConnectedApps() {
```

Comment:
fetchConnectedApps and revokeConnectedApp doesn't need to be in the client-store anymore either

Reply:
Sure, I move them to the local file.

Reply:
nice